README.md

```diff
@@ -240,12 +252,15 @@ does not reflect common usage.
 `prefer_foreach`
 is too strict; `forEach` is not always an improvement.
 
+`prefer_if_elements_to_conditional_expressions`
+users prefer `?:` over `if`/`else`.
+
 `prefer_int_literals`
 does not reflect common usage.
 
 `prefer_typing_uninitialized_variables`
-will be obsoleted by NNBD, which comes with type inference for uninitialized
-variables.
+will be obsoleted by sound null safety, which comes with type inference for
+uninitialized variables.
```

Comment:
This turned out to be wrong!... please remove from the banned list :)

Reply:
Done.